app/talk/index.cjsx

```diff
@@ -29,5 +29,5 @@ module?.exports = React.createClass
         </button>
       </form>
 
-      <RouteHandler {...@props} section={@state.currentSection} />
+      <RouteHandler {...@props} section={currentSection(@props.project)} />
```

Comment:
Couldn't this just be `<RouteHandler {...@props} section="zooniverse" />`? And then have the project-specific Talk instance use the `lib/project-section` function directly?